src/main/java/javaserver/ResponseBuilders/DirectoryResponseBuilder.java

```diff
@@ -0,0 +1,48 @@
+package javaserver.ResponseBuilders;
+
+import java.util.Arrays;
+
+import javaserver.HTMLContent;
+import javaserver.HTTPStatusCode;
+import javaserver.Request;
+import javaserver.Response;
+import javaserver.Routes;
+
+public class DirectoryResponseBuilder implements ResponseBuilder {
+
+	private Response response;
+	private String directory = "public";
+	
+	@Override
+	public Response getResponse(Request request) {
+		this.response = new Response();
+		setResponseData(request);
+		return this.response;
+	}
+
+	@Override
+	public void setResponseData(Request request) {
```

Comment:
It doesn't look like you're using this `setResponseData` outside of these objects. Any reason why you're exposing them?